src/pbview.cpp

```diff
@@ -285,8 +285,11 @@ void PbView::handle_resize()
 
 void PbView::apply_colors_to_all_forms()
 {
-	colorman.apply_colors(dllist_form);
-	colorman.apply_colors(help_form);
+	using namespace std::placeholders;
+	colorman.apply_colors(std::bind(&newsboat::Stfl::Form::set, &dllist_form, _1,
+			_2));
+	colorman.apply_colors(std::bind(&newsboat::Stfl::Form::set, &help_form, _1,
+			_2));
```

Comment:
I'm okay with keeping `std::bind` here, but please avoid it in the future as it has [plenty of pitfalls](https://abseil.io/tips/108). Lambdas are usually sufficient.

Reply:
I wasn't aware of those pitfalls, thanks, thanks!